homeassistant/components/light/limitlessled.py

```diff
@@ -33,32 +37,54 @@
 from homeassistant.util.color import color_RGB_to_xy
 
 _LOGGER = logging.getLogger(__name__)
-REQUIREMENTS = ['ledcontroller==1.0.7']
+REQUIREMENTS = ['ledcontroller==1.1.0']
 
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the LimitlessLED lights. """
     import ledcontroller
 
-    led = ledcontroller.LedController(config['host'])
+    # Handle old configuration format:
+    bridges = []
```

Comment:
This code is fine but can be more Pythonic. Using the `get` method on a dict will allow you to specify a value if the key does not exist. Means this code can be rewritten:
```python
bridges = config.get('bridges', [config])
```